js/controller.js

```diff
@@ -192,6 +192,21 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     onAuthorizationFetched: function(event, authorization) {
       this.state.authorization = authorization;
+
+      // Outside of state check as this is needed in order to be able to perform the state check.
+      this.mb.subscribe(OO.EVENTS.PLAYBACK_READY, 'customerUi', _.bind(this.onPlaybackReady, this));
```

Comment:
I still don't think this is the right approach. I have couple suggestions:
1. Seems like there are couple places that need to subscribe / unsubscribe to the same events. let's just refactor them into 2 methods: subscribeBasicPlaybackEvents() and unsubscribeBasicPlaybackEvents()
2. Rather than listening to authorization fetched, perhaps you should listen to EMBED_CODE_CHANGED event. Then calls subscribeBasicPlaybackEvents().
3. We can move the same events inside init() to be listened in point number 2
4. Find a way to make sure that there is no duplicate subscription. You can use a flag by utilizing this.state